cmd/server-mux.go

```diff
@@ -457,6 +460,8 @@ func (m *ServerMux) ListenAndServe(certFile, keyFile string) (err error) {
 			// Execute registered handlers, update currentReqs to keep
 			// tracks of current requests currently processed by the server
 			atomic.AddInt32(&m.currentReqs, 1)
+			lmsg := fmt.Sprintf("serving for %s", r.URL.String())
+			log.logger.Info(lmsg)
```

Comment:
In `NewLogger()`, you set to `logrus.ErrorLevel`.  There is no way you can get this log printed.

As there is no log level setting controlled by user, these logs are never known.